.storybook/main.ts

```diff
@@ -10,7 +10,7 @@ const mainAppStories = [
 
 const config: StorybookConfig = {
   stories: mainAppStories,
-  staticDirs: ["../resources/frontend_client"],
+  staticDirs: ["../resources/frontend_client", "../.storybook-sdk/msw-public"],
```

Comment:
Not `sdk` right? We can practically name this anything like
https://github.com/metabase/metabase/blob/eaa76cab4aaf4516084e08e0b738ee05525c1cc1/.storybook-sdk/main.js#L16

If we wanna make it similar to the one I added to the SDK then
```suggestion.
  staticDirs: ["../resources/frontend_client", "./msw-public"],
```

Reply:
changed